library/src/main/java/org/xmtp/android/library/push/README.md

```diff
@@ -0,0 +1,79 @@
+# Enable your app to send push notifications
+
+You can use the `xmtp-android` SDK and a notification server to enable your app to send push notifications. This tutorial describes the process using Firebase Cloud Messaging and an example notification server written in Golang.
+
+## Set up a Firebase Cloud Messaging project
+
+For this tutorial, we'll use [Firebase Cloud Messaging](https://console.firebase.google.com/) (FCM) as a convenient way to set up a messaging server.
+
+1. Create an FCM project.
+
+2. Add an app to your FCM project. This generates a `google-services.json` file that you need in subsequent steps.
+
+3. Add the `google-services.json` file to your Android project as described in the FCM project creation process.
+
+4. Generate your FCM credentials, which you need to run your notification server. To do this, from your FCM dashboard, click the gear icon next to **Project Overview** and select **Project settings**. Select **Service accounts**. Select **Go** and click **Generate new private key**. 
+
+## Run an example notification server
+
+Now that you have your FCM server set up, take a look at the [export-kotlin-proto-code](https://github.com/xmtp/example-notification-server-go/tree/np/export-kotlin-proto-code) branch in the `example-notifications-server-go` repo. 
+
+This example branch can serve as the basis for what you might want to provide for your own notification server. The branch also demonstrates how to generate the proto code.
+
+Here are the steps to run a notification server based on the example branch:
+
+1. Clone the [example-notification-server-go](https://github.com/xmtp/example-notification-server-go) repo.
+
+2. Complete the steps in [Local Setup](https://github.com/xmtp/example-notification-server-go/blob/np/export-kotlin-proto-code/README.md#local-setup).
+
+3. Get the FCM project ID and `google-services.json` file you created earlier and run:
+
+    ```bash
+    dev/run \                                                                                                                                                                                                                              ✘ 1
+    --xmtp-listener-tls \
+    --xmtp-listener \
+    --api \
+    -x "production.xmtp.network:5556" \
+    -d "postgres://postgres:xmtp@localhost:25432/postgres?sslmode=disable" \
+    --fcm-enabled \
+    --fcm-credentials-json=YOURFCMJSON \
+    --fcm-project-id="YOURFCMPROJECTID"
+    ```
+
+4. You should now be able to see push notifications coming across your local network.
```

Comment:
I turned this line into a step - and then made the bullets below it subsequent steps. Is this an accurate portrayal? Or should this signify a new procedure and set of steps?